mitodl/bootcamp-ecommerce
# Add support for uploading a resume to an existing application

#### What are the relevant tickets?
Fix #448

#### What's this PR do?
Adds a new field for resume upload date, adds support for saving a resume file, with the allowed extension, and triggers a transition in state for the application.

#### How should this be manually tested?
Run migrations, create a bootcamp application with various states, create a file with various file extensions, and call `process_upload_resume`. If the file is valid the `BootcampApplication.status` should change.
